components/tokens/custom-spectrum/custom-vars.css

```diff
@@ -35,16 +35,13 @@ governing permissions and limitations under the License.
   --spectrum-animation-ease-out: cubic-bezier(0, 0, 0.40, 1);
   --spectrum-animation-ease-linear: cubic-bezier(0, 0, 1, 1);
 
-  --spectrum-sans-serif-font: var(--spectrum-sans-serif-font-family);
```

Comment:
Do we want to add backwards compatibility here by remapping `--spectrum-sans-serif-font` to `--spectrum-sans-font-family-stack`? My thought was in case someone was using the other variable directly.

Reply:
Yeah, this is a good suggestion. I'll update.